pkg/controller/nodelifecycle/metrics.go

```diff
@@ -30,6 +30,9 @@ const (
 	zoneNoUnhealthyNodesKey = "unhealthy_nodes_in_zone"
 	evictionsNumberKey      = "evictions_number"
 	evictionsTotalKey       = "evictions_total"
+
+	updateNodeHealthKey     = "update_node_health_duration_seconds"
```

Comment:
Those names looks reasonable to me now - @dgrisonnet - are you ok with the new ones?

Reply:
Yes, it is easier to notice the difference now :+1: